libcnb-data/src/newtypes.rs

```diff
@@ -1,4 +1,8 @@
-/// Macro to generate newtypes backed by `String`
+// The compiler has a hard time figuring out that we actually use libcnb_sys in here. To avoid
+// false-positive warnings about unused crates, we import it as _ here:
+use libcnb_proc_macros as _;
```

Comment:
This doesn't appear to be needed any more. Perhaps the import in `libcnb-data/src/internals.rs` meant it's no longer needed?